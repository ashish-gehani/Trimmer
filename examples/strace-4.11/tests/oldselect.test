#!/bin/sh

# Check old select syscall decoding.

. "${srcdir=.}/init.sh"

run_prog
run_strace -a34 -eselect $args
match_diff

exit 0
