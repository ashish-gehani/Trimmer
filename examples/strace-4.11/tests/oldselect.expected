select(2, [1], [0], NULL, {0, 42}) = 0 (Timeout)
+++ exited with 0 +++
